setup.py

```diff
@@ -19,6 +19,7 @@ def read_file(file):
       license='MIT',
       zip_safe=False,
       keywords='gis geospatial geographic shapefile shapefiles',
+      python_requires='>= 2.7, >= 3',
```

Comment:
If I read the docs correctly, this would be interpreted as "Python 3 only" in the end. https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

I think, actually `python_requires='>=2.7',` is what we want? Even though to a human reader intuitively it looks like Python 2 only.. (or maybe `python_requires='>=2.7, <4',` to make this more clear?)

Reply:
Adding an upper limit should probably be done only after it is know not to work.